web/src/lib/roadmap/roadmapBuilder.test.ts

```diff
@@ -33,6 +33,30 @@ describe("roadmapBuilder", () => {
     ).toEqual(expect.arrayContaining(["tea-task-1-id"]));
   });
 
+  it("builds a roadmap with license add-ons only when industry is undefined", async () => {
+    const roadmap = await buildRoadmap({
+      industryId: undefined,
+      addOns: ["license-addon"],
+    });
+    expect(
+      roadmap.tasks.map((it) => {
+        return it.id;
+      })
+    ).toEqual(expect.arrayContaining(["license-task-1-id"]));
+  });
+
+  it("prioritizes a task over a license when building a roadmap with add-ons only when industry is undefined", async () => {
+    const roadmap = await buildRoadmap({
+      industryId: undefined,
```

Comment:
[Dust] I'm a little confused about this test. The reason for saying this is that I think a license task would be tied to the industry. If you have no industry, how could you possibly have a license task? Am I off? I might be off. I think this test is OK, I don't know if we need another.

Reply:
So I'm not super familiar with the section of the app to be honest, and the way the code is written just for roadmapBuilder it looks like that is a possible state, but if you're saying that from a logical perspective it shuldn't happen in the app when the users are taking their flows that makes sense. 

I just defined the industry ID and moved the test down a bit.